docs/scripts/utils.py

```diff
@@ -109,7 +112,10 @@ def create_example_documentation(example_dest_dir: str, ignore_examples: bool):
         for file in (inner_pbar := tqdm(py_files, leave=False)):
             # Include the name of the file to the toctree
             # Format it by replacing underscores and capitalizing the words
-            file_name = file.stem
+            file_name, file_ending = file.stem, file.suffix
+            # If we happen to collect anything else then a python file, we continue.
+            if not file_ending == ".py":
+                continue
```

Comment:
Why is this needed? You have the `glob` above!?

Reply:
Because reading code is hard, writing redundancy is not 😆  Jokes aside, you are fully correct, this is not needed and I will remove this.